### PR TITLE
Update date picker and dropdown elements following work in roadrunner

### PR DESCRIPTION
**Pull request by learykara**
@evanrs  please take a look when you have a minute. Minor changes to style, default props, etc in date picker and dropdown

### CODE REVIEW

File: src/DatePicker/DatePicker.es6 [hunk 1]
```diff
@@ -80,12 +84,13 @@ class DatePicker extends React.Component {
                     dayOfYear: moment(startDay).add(day, 'day').dayOfYear()
                   }});
 
-    activeIndex = activeIndex ||
-      _.findIndex(days, {dayOfYear: moment().dayOfYear()});
+    activeIndex = (! activeIndex || activeIndex === -1) ?
+      _.findIndex(days, {dayOfYear: moment(defaultDate || '').dayOfYear()}) : activeIndex;
```
Comment (evanrs) [inline src/DatePicker/DatePicker.es6]: The else clause of the ternary expression should sit on the next line when following a calculated value

File: src/Dropdown/Dropdown.jsx [hunk 2]
```diff
@@ -16,6 +16,7 @@ const Dropdown = React.createClass({
 
   propTypes: {
     data: React.PropTypes.array.isRequired,
+    initialSelectedInd: React.PropTypes.number,
```
Comment (evanrs) [inline src/Dropdown/Dropdown.jsx]: Full words are preferable, perhaps `defaultIndex` and `activeIndex`?

File: src/Dropdown/Dropdown.jsx [hunk 3]
```diff
@@ -16,6 +16,7 @@ const Dropdown = React.createClass({
...
     selectedInd: React.PropTypes.number,
     defaultDisplay: React.PropTypes.string,
     handleChange: React.PropTypes.func.isRequired
```
Comment (evanrs) [inline src/Dropdown/Dropdown.jsx]: The api for event handlers usually has `on` as prefix, so `onMouseIn` or `onChange`.

File: src/Dropdown/Dropdown.jsx [hunk 4]
```diff
@@ -72,15 +74,19 @@ const Dropdown = React.createClass({
...
-    let {selectedInd, data, defaultDisplay} = this.props;
+    let {
+      initialSelectedInd, selectedInd, data,
+      defaultDisplay, className} = this.props;
 
-    const buttonClasses = classnames('button', 'button__white', 'dd-button');
-    const dropdownClasses = classnames(
+    selectedInd = selectedInd || initialSelectedInd;
+
+    const buttonClasses = cx('button', 'button__white', 'dd-button');
```
Comment (evanrs) [inline src/Dropdown/Dropdown.jsx]: `cx` isn't need here because it's no removing any falsey values or helping to concatenate variables.

**Comment by evanrs** [comment]
Take a look at the comments, but otherwise solid :+1:

File: src/Dropdown/Dropdown.jsx (hunk 2 above)
Comment (learykara) [inline src/Dropdown/Dropdown.jsx]: reluctant to change this now as there are now a few different apps using the dropdown and I'd have to go back and change individually in all of them, but will keep this in mind going forward

File: src/Dropdown/Dropdown.jsx (hunk 3 above)
Comment (learykara) [inline src/Dropdown/Dropdown.jsx]: ditto above

**Pull request merged**